sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -71,7 +71,7 @@ Azure Active Directory (AAD) authentication is an optional feature that can be u
 
 ```C#
 // Call AddAzureMonitor and set Credential to authenticate through Active Directory.
```

Comment:
```suggestion
// Call WithAzureMonitor and set Credential to authenticate through Active Directory.
```

Reply:
Adding to this, there are several references to `AddAzureMonitor` in this file. Need to do a find/replace.